docs/development.rst

```diff
@@ -5,9 +5,9 @@
 Development
 ===========
 
-This chapter will get you started with |project_name| development.
+This chapter will get you started with Borg development.
```

Comment:
are this kind of changes present in other 1.1-maint docs files? if not, it should not be here either.

Reply:
You're right, the |project_name| thing is used everywhere. I will undo that change.